### PR TITLE
Add dependabot configuration file

### PR DESCRIPTION
**Pull request by vsergiu93**
Added a dependabot.yml configuration file to enable automatic dependency updates for npm and composer packages. This will ensure the application stays secure and up-to-date without manual intervention.

### CODE REVIEW

File: .github/dependabot.yml [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+    schedule:
+      interval: "weekly"
+
+  # Enable version updates for composer
+  - package-ecosystem: "composer"
+    # Look for `composer.json` and `composer.lock` files in the `root` directory
+    directory: "/"
+    # Check for updates once a week
+    schedule:
+      interval: "weekly"
```
Comment (Plytas) [inline .github/dependabot.yml:17]: Propose to add labels and a check for actions.

```suggestion
  - package-ecosystem: "npm"
    # Look for `package.json` and `lock` files in the `root` directory
    directory: "/"
    # Check for updates once a week
    schedule:
      interval: "weekly"
    labels:
      - "dependencies"

  # Enable version updates for composer
  - package-ecosystem: "composer"
    # Look for `composer.json` and `composer.lock` files in the `root` directory
    directory: "/"
    # Check for updates once a week
    schedule:
      interval: "weekly"
    labels:
      - "dependencies"
      
  - package-ecosystem: "github-actions"
    directory: "/"
    schedule:
      interval: "weekly"
    labels:
      - "dependencies"
```

**Review by Plytas — CHANGES_REQUESTED** [review]

**Comment by brendt** [comment]
Ping me when this is ready to merge :)

File: .github/dependabot.yml (hunk 1 above)
Comment (vsergiu93) [reply .github/dependabot.yml]: Thank you

**Comment by vsergiu93** [comment]
@brendt From my side is ok, I added the requested change :).

**Comment by brendt** [comment]
I'll fix PHPStan on my end :)

**Comment by brendt** [comment]
Thanks, btw :)

**Comment by Plytas** [comment]
So it begins 😄 
image

**Pull request merged**